src/BIP44Node.ts

```diff
@@ -161,24 +156,10 @@ export class BIP44Node implements BIP44NodeInterface {
       });
     }
 
-    const {
-      privateKey,
-      publicKey,
-      chainCode,
-      depth,
-      parentFingerprint,
-      index,
-    } = options;
-
-    validateBIP44Depth(depth);
+    validateBIP44Depth(options.depth);
 
     const node = await SLIP10Node.fromExtendedKey({
-      privateKey,
-      publicKey,
-      chainCode,
-      depth,
-      parentFingerprint,
-      index,
+      ...options,
```

Comment:
What's the reasoning for being less explicit here now?

Reply:
Just seemed easier to read, but I can revert if you prefer.

Reply:
I think in this specific case its probably better to be explicit